src/components/Forms/LoginForm.js

```diff
@@ -59,7 +59,7 @@ const LoginForm = ({ handleChange, data, onSubmit, loading }) => (
       <Button
         block
         disabled={loading}
-        onPress={onSubmit}
+        onPress={() => onSubmit()}
```

Comment:
is there a reason for doing this?

Reply:
@edencakir yes. just `onPress={onSubmit}` would send `event` as a first parameter, which is considered as `truthy` value. This makes sure onSubmit is called without any params.

Reply:
@BilalBudhani  Ah I see, never thought this that way